website/content/en/docs/upgrading-sdk-version/v1.10.0.md

```diff
@@ -0,0 +1,12 @@
+---
+title: v1.10.0
+weight: 998990000
+---
+
+## (ansible/v1) Bump Kustomize version in helm project scaffolding to 3.8.7
+
+Ansible projects are now scaffolded with `Kustomize` version `3.8.7`. This version of kustomize requires that the path to patch file be provided with the `--path` flag in `add patch` command. In `molecule/default/prepare.yml` file, update the command under `Set pull policy`, to be: ```diff
+     - name: Set pull policy
+-      command: '{{ "{{ kustomize }}" }} edit add patch pull_policy/{{ "{{ operator_pull_policy }}" }}.yaml' +      command: '{{ "{{ kustomize }}" }} edit add patch --path pull_policy/{{ "{{ operator_pull_policy }}" }}.yaml' ```
```

Comment:
```suggestion
Ansible projects are now scaffolded with `Kustomize` version `3.8.7`. This version of kustomize requires that the path to patch file be provided with the `--path` flag in `add patch` command. In `molecule/default/prepare.yml` file, update the command under `Set pull policy`, to be: 
```diff
     - name: Set pull policy
-      command: '{{ "{{ kustomize }}" }} edit add patch pull_policy/{{ "{{ operator_pull_policy }}" }}.yaml' 
+      command: '{{ "{{ kustomize }}" }} edit add patch --path pull_policy/{{ "{{ operator_pull_policy }}" }}.yaml' ```
```
this should be appearing as a diff. maybe formatting issue